apps/omg_watcher/lib/omg_watcher/api/utxo.ex

```diff
@@ -41,7 +45,7 @@ defmodule OMG.Watcher.API.Utxo do
   end
 
   @spec compose_utxo_exit(Utxo.Position.t()) :: {:ok, exit_t()} | {:error, :utxo_not_found}
-  def compose_utxo_exit(utxo_pos) when is_deposit(utxo_pos) do
+  def compose_utxo_exit({:utxo_position, blknum, _, _} = utxo_pos) when is_deposit(blknum) do
```

Comment:
verbose use of a `{:utxo_position, ...}` should be avoided in favor of `Utxo.position(...)`, even if this is an intermediate result.